.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+#      - name: Cache Gradle
```

Comment:
with caches the gradle build fails in test

Reply:
or maybe there is another flakyness in tests

Reply:
Use the https://github.com/marketplace/actions/gradle-command action, which does caching out of the box.

Reply:
i dont know how to marry this with the two gradle commands

Reply:
the caching seems to lead to constant test errors and not only erratic ones

Reply:
I don't understand the problem, can you elaborate?

Reply:
there where test fails in 10/10 builds
without case in 1/10

Reply:
I'm confused, do you mean your own caching that you tried or the gradle-command action?

Reply:
this is the caching as documented in the official docs.
i dont know how to solve the release 
problem from below with 
gradle-command 

do you want to take over PR?
i just want to get this shit done.
i have zero github actions knowledge

Reply:
> i dont know how to solve the release problem from below with gradle-command

Both only need one Gradle command. The first does `check` (always), the second does `release` (only on tags). So you just replace `run` with `gradle-command`

Reply:
????????
i dont understand

Reply:
See the change I just pushed.

Reply:
Will the gradle release pick up from the check and reuse the Dämon even if they are in two steps
As I can’t test I splittre the steps to be sure

Reply:
Yes, it will. That's what I meant by "it's the same".